client/src/components/Sidebar/menu/SubMenuItem.tsx

```diff
@@ -88,6 +88,7 @@ export const SubMenuItem: React.FunctionComponent<SubMenuItemProps> = (
         <ul aria-labelledby={`wagtail-sidebar-submenu${path.split('.').join('-')}-title`}>
           {renderMenu(path, item.menuItems, slim, state, dispatch, navigate)}
         </ul>
+        {item.footerText && <p className="sidebar-sub-menu-panel__footer">{item.footerText}</p>}
```

Comment:
question: should there be a limit for the footer text? how would this look with a long phrase?

Reply:
It should wrap on to multiple lines if it's too long